_posts/ru/2020-09-18-ALTF4-5.md

```diff
@@ -0,0 +1,103 @@
+---
+author: pocarski, Romner, Dr.MagPie
+editor: Therenas, nicgarner
+translator: ftvkyo
+image:
+  url: /assets/ALTF4/5/thumbnail.jpg
+  transcription: Fusion rocket exploding
+title: "Alt-F4 #5 - Improving on Perfection"
```

Comment:
Line 7-8 need translation

Reply:
I see, thanks. When I looked for a reference I happened to look at posts that didn't translate these lines. I'll update them.

Reply:
correct, some posts in russian seems to lack these being translated. you can work on it if you want :)